app/views/users/logout.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.code 200
```

Comment:
we hardcoded  code = 200 everywhere. what purpose does this serve? We should find a way to clean this up, waiting for your proposal :)